package.json

```diff
@@ -5,17 +5,14 @@
   "description": "The BRP application",
   "scripts": {
     "start": "node .",
-    "start:dev":
-      "./node_modules/parallelshell/index.js 'DEBUG=* ./node_modules/nodemon/bin/nodemon.js .' 'npm run watch:scss'",
+    "start:dev": "./node_modules/parallelshell/index.js 'DEBUG=* ./node_modules/nodemon/bin/nodemon.js .' 'npm run watch:scss'",
     "watch:scss": "./node_modules/nodemon/bin/nodemon.js -e scss -x 'npm run sass'",
     "test": "NODE_ENV=test mocha",
     "lint": "./node_modules/eslint/bin/eslint.js .",
     "style": "./node_modules/jscs/bin/jscs **/*.js --config=./.jscsrc.json",
-    "quality":
-      "./node_modules/plato/bin/plato -r -x 'node_modules|reports|test' -d reports/plato .",
+    "quality": "./node_modules/plato/bin/plato -r -x 'node_modules|reports|test' -d reports/plato .",
```

Comment:
I mean if they are installed globally and not defined as dependencies.
However, I'm assuming what you are getting at it that they *should* be defined as dependencies, and once they are, "any locally installed module with a defined binary can be used directly from within an npm script."